Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-06-05
Forwarded: 2012-06-05
Description: prevent FTBFS for 'trunk' versions after 2012-05-01
 AbiGrammar.cpp: In member function 'virtual bool AbiGrammar::notify(AV_View*, AV_ChangeMask, void*)':
 AbiGrammar.cpp:106:18: warning: variable 'pText' set but not used [-Wunused-but-set-variable]
 AbiGrammar.cpp: In function 'int abi_plugin_register(XAP_ModuleInfo*)':
 AbiGrammar.cpp:159:38: error: cannot allocate an object of abstract type 'AbiGrammar'
 AbiGrammar.cpp:65:7: note:   because the following virtual functions are pure within 'AbiGrammar':
 In file included from ../../../src/af/xap/xp/xap_App.h:39:0,
                  from AbiGrammar.cpp:34:
 ../../../src/af/xap/xp/xav_Listener.h:110:15: note:     virtual bool AV_ListenerExtra::notifyWithArg(AV_View*, AV_ChangeMask, void*)
 make[5] *** [AbiGrammar.lo] Error 1


--- a/plugins/grammar/xp/AbiGrammar.cpp
+++ b/plugins/grammar/xp/AbiGrammar.cpp
@@ -92,7 +92,7 @@
 		  return false;
 		}
 
-virtual bool notify(AV_View * pAView, const AV_ChangeMask mask, void * pPrivateData)
+virtual bool notifyWithArg(AV_View * pAView, const AV_ChangeMask mask, void * pPrivateData)
 		{
 		  if(mask != AV_CHG_BLOCKCHECK)
 		  {
